static/src/javascripts-legacy/projects/common/modules/identity/autosignin.js

```diff
@@ -52,24 +52,22 @@ function (
         };
 
         this.signin = function (authResponse, name) {
-            ajax({
-                url: config.page.idWebAppUrl + '/jsapi/facebook/autosignup',
-                cache: false,
-                crossOrigin: true,
+            fetch(config.page.idWebAppUrl + '/jsapi/facebook/autosignup', {
+                mode: 'cors',
                 type: 'jsonp',
-                data: {
+                body: {
                     signedRequest: authResponse.signedRequest,
                     accessToken: authResponse.accessToken
-                },
-                success: function (response) {
-                    self.welcome(name);
-                    if (response.status === 'ok') {
-                        var profile = new Profile({
-                            url: config.page.idUrl
-                        });
-                        profile.init();
-                        new Toggles().init();
-                    }
+                }
+            }).then(function(response) {
+                self.welcome(name);
+
+                if (response.status === 'ok') {
```

Comment:
The fetch docs state that `status` is always the HTTP code (https://developer.mozilla.org/en-US/docs/Web/API/Response/status). I guess our implementation is different, because we are passing through the `reqwest.status` ... ?